db/contacts.json

```diff
@@ -0,0 +1,94 @@
+{
+  "contacts": [
+    {
+      "firstName": "Alejandro",
+      "lastName": "Sanchez",
+      "preferedName": "Flex Sanchez",
+      "email": "alesanchez728@gmail.com",
+      "primaryPhone": "(708)-307-1983",
+      "address": "1500A Norvel Ave. Nashville, TN",
+      "birthday": "10-03-1995"
+    },
+    {
+      "firstName": "Dummy",
+      "lastName": "McDumbface",
```

Comment:
Make these guys unique (at least somewhat) so you can make sure that stuff is working later. 

ALSOM YOU'RE A DUMMY MCDUMBFACE